distribution/openrct2.d.ts

```diff
@@ -1386,6 +1387,10 @@ declare global {
         readonly crashIntoType: VehicleCrashIntoType;
     }
 
+    interface CalculateGuestCapArgs {
+        suggestedMaxGuests: number;
```

Comment:
Could you add documentation here that it's possible to assign this one to override the park's suggested guest cap?